AlphaWallet/Tokens/Views/TokensViewControllerTableViewSectionHeader.swift

```diff
@@ -9,27 +9,67 @@ extension TokensViewController {
 
         var filterView: SegmentedControl? {
             didSet {
-                guard let filterView = filterView else {
+                guard let filterView = self.filterView else {
                     if let oldValue = oldValue {
                         oldValue.removeFromSuperview()
                     }
                     return
                 }
+                
                 filterView.translatesAutoresizingMaskIntoConstraints = false
-                contentView.addSubview(filterView)
+                self.containerView.addArrangedSubview(filterView)
+                
                 NSLayoutConstraint.activate([
-                    filterView.anchorsConstraint(to: contentView, edgeInsets: .init(top: 0, left: 0, bottom: 7, right: 0)),
+                    filterView.widthAnchor.constraint(equalTo: self.widthAnchor),
```

Comment:
Doesn't this change cause the horizontal and vertical position to be undefined? Unless it's anchored somewhere else?

Reply:
No, it should work ok, for vertical and horizontal layouts

Reply:
Oh, you are right. I missed that it is added to a vertical `UIStackView`